boards/arm/npcx4m8f_evb/npcx4m8f_evb.dts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nuvoton/npcx4m8f.dtsi>
+#include "npcx4m8f_evb-pinctrl.dtsi"
+
+/ {
+	model = "Nuvoton NPCX4M8F evaluation board";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart1;
+		zephyr,flash = &flash0;
+		zephyr,keyboard-scan = &kscan_input;
+	};
+
+	aliases {
+		/* For samples/basic/blinky_pwm */
+		pwm-led0 = &pwm_led0_green;
+		/* For gpio test suites */
+		led0 = &gpio_led_red;
+		/* For pwm test suites */
+		pwm-0 = &pwm6;
+		/* For i2c test suites */
+		i2c-0 = &i2c0_0;
+		/* For watchdog sample */
+		watchdog0 = &twd0;
+		peci-0 = &peci0;
+		/* For samples/drivers/spi_flash */
+		spi-flash0 = &int_flash;
+		/* For kscan test suites */
+		kscan0 = &kscan_input;
+	};
+
+	leds-pwm {
+		compatible = "pwm-leds";
+		pwm_led0_green: pwm_led_0 {
+			pwms = <&pwm6 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+			label = "User D7 green";
+		};
+	};
+
+	leds-gpio {
+		compatible = "gpio-leds";
+		gpio_led_red: led_0 {
+			gpios = <&gpio6 0 GPIO_ACTIVE_LOW>;
+			label = "User D8 red";
+		};
+	};
+};
+
+/* Overwrite default device properties with overlays in board dt file here. */
+&uart1 {
+	status = "okay";
+	current-speed = <115200>;
+	/* Use UART1_SL2 ie. PIN64.65 */
+	pinctrl-0 = <&uart1_2_sin_gp64
+		     &uart1_2_sout_gp65>;
```

Comment:
this (and I think few others) fits well within 80 columns, can you unwrap it?

Reply:
Done.